src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -206,7 +340,40 @@ public static void setMythicVariableMap(Entity entity, Map<String, Variable> map
         registry.putAll(map);
     }
 
+    public static LocationTag locationFromAbstractLocation(AbstractLocation location) {
+        return new LocationTag(location.getWorld().getName(), location.getX(), location.getY(), location.getZ(), location.getYaw(), location.getPitch());
```

Comment:
`BukkitAdapter#adapt(AbstractLocation)` - can use that directly instead of this method